manifests/profiles/default.yaml

```diff
@@ -251,10 +251,8 @@ spec:
       enabled: true
       v2:
         enabled: true
-        metadataExchange:
-          wasmEnabled: true
+        metadataExchange: {}
```

Comment:
@rshriram instead of reverting the PR, you can just change the wasEnabled: false
 And it will have the same effect. I suggest you do that.

Reply:
i have no clue about layout of yamls.. To me, its easier to revert than understand and turn off configs..

Reply:
Change 2 occurrences of

wasmEnabled: true

To 

wasmEnabled: false 

and nothing else.